src/EventListener/InitializeSystemListener.php

```diff
@@ -138,10 +138,12 @@ public function onConsoleCommand()
     protected function setConstants($mode, $route)
     {
         // The constants are deprecated and will be removed in version 5.0.
+        $time = microtime(true);
```

Comment:
Ok. In the meantime, please revert this change, since it is not related to the `DEPRECATED.md` file.